.github/actions/sourceclear/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh -l
+echo "Download srcclr ci script ..."
+curl -sSL https://download.sourceclear.com/ci.sh --output srcclr.sh
```

Comment:
Not too dangerous to curl this?

Reply:
Why?

Reply:
Just about directly down a third party .sh to execute it, but  it's not really relevant here

Reply:
I took the example from sourceclear repo, and adapted it to our repo/need.
The curl is from here : https://github.com/srcclr/example-github-actions/blob/master/.github/actions/sourceclear/entrypoint.sh

With this, we download the official sourceclear script, and it's always up to date.

The only risk I see is if sourceclear get hacked and the file is replaced by something malicious.
So our solutions:
- go with this solution, the risk is limited and not really important since the repo is open sourced.
- download the current version of the script, that should not change a lot, and push it in our repo. And if it requires a change, we'll do a PR.

Either way I think it's acceptable.
Any preference @vbocquet @frassinier ?

Reply:
I'm ok with you in you're first solution, the risk is limited 👍

Reply:
And you're right, we have an open source repo